utils/statserial/patches/800-fix_compilation.patch

```diff
@@ -0,0 +1,22 @@
+--- a/Makefile
```

Comment:
This patch can not be accepted as it is. 

Why do we need to have it? There are formal issues (author, description, subject), etc.

Reply:
This patch is required for multiple reasons:

- basic Makefile overrides CC which does not work for cross-compilation
- libcurses are named incorrectly for current naming of libncurses
- using LD directly instead of CC does cause some errors when linking on some platforms, it is more universal that way